src/components/Accounts/ChargeFeeFine/ChargeForm.js

```diff
@@ -218,10 +221,11 @@ class ChargeForm extends React.Component {
         </Button>
         <Button
           id="chargeOnly"
-          disabled={pristine || submitting || !valid}
+          disabled={pristine || this.state.submitting || !valid}
           onClick={() => {
             change('pay', false);
             handleSubmit(data => onSubmit(data));
+            this.setState({ submitting: true });
```

Comment:
As I see we can't submit again without leaving the page of refresh. Should we set `submitting` flag back to `false` after  submit was completed?

Reply:
I guess here we do not need it because after submit we will go to the one route back [here].(https://github.com/folio-org/ui-users/blob/bce2383f54b021a20934b9880d95877d1f7f31ee/src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js#L408)

Reply:
It's not a good practice to rely on another component in such cases. If onSubmit implementation will be changed -  we will receive a bug.

Reply:
I don't entirely understand the flow here, nor why the previous implementation that relied on `props.submitting` was insufficient. In other words, I would expect the `disabled` implementation for both the "Charge and pay" and the "Pay only" buttons to be the same. Can you explain, @annamelnyk, why they are different?

Reply:
(Related since you referenced it, but not really related to this PR: using `history.goBack()` in [`ChargeFeeFine.js`](https://github.com/folio-org/ui-users/blob/bce2383f54b021a20934b9880d95877d1f7f31ee/src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js#L408) feels like the wrong implementation. We might indeed want to process the action and then "return" to the "previous" page, but only in the sense that we are visiting a given location a second time. I would expect `history.push(that previous location)`. That function could really use some comments too, or a reference to a wiki page or a ticket or something explaining the overall flow of the processes for charge, charge and pay, etc. Do you know if we have that kind of documentation anywhere?)

Reply:
@zburke as I know there no any documentation exist related to `Fee/fines`, I will ask Holly about  `history.goBack()`.

Reply:
Holly agreed to remove history.goBack(). Also with `props.submitting` is possible to click multiple times on button, so I created extra `formSubmitting` flag, and  added disabled implementation for both the "Charge&pay now" and the "Charge only"  buttons